### ISSUE TITLE
Ability to set reminder to pray for something specific for a person

### ISSUE THREAD

**Issue by LIL717** [ceaseless-prayer/CeaselessIOS#156]
A friend was telling me about an interview she has on Tuesday.  I entered the prayer request in the Prayer Journal, but was thinking that it would be nice to have a reminder to pray for her at 10 on Tuesday during the interview.  Maybe adding a button to the Notes View to add a reminder.  The reminder can be added to the Reminders on the device so it does not need to be managed by Ceaseless.

Here is an example of a simple screen to add a reminder.

![img_7391](https://cloud.githubusercontent.com/assets/5344311/7569902/00a63340-f7c1-11e4-9af8-85937bc22981.PNG)

**Comment by chrislim** [comment]
That would be cool! Could the reminder have an "app link" that would take them to the contact in Ceaseless?

**Comment by LIL717** [comment]
I don't think so, but maybe we could just build our own notification like we do for the daily reminder

**Comment by chrislim** [comment]
Yea we could do that. It makes sense--more of a natural fit for people wanting to set a follow up date.

**Comment by chrislim** [comment]
How would we add this ability to the Notes UI? Would we add a menu somewhere or shortcuts? Or a "..." more button?

**Comment by chrislim** [comment]
We can add it to the notes UI by making the People Tagger less wide and having a Reminder icon button next to it. When the user taps that icon, a view controller is presented modally to let them pick when they want to be reminded. It could also have shortcuts like "in a week" or "in a month". Then they can save the reminder.

If they tap the reminder icon when a reminder has already been set, then the modal will let them edit the reminder or show a button that lets them delete it.

The reminder can be implemented with a local notification.
